src/manipulation.js

```diff
@@ -201,7 +201,7 @@ jQuery.extend({
 		return clone;
 	},
 
-	buildFragment: function( elems, context, scripts, selection ) {
+	buildFragment: function( elems, context, scripts, selection, ignored ) {
```

Comment:
In short, because of self-manipulation. `.replaceWith` is the worst offender, but it's basically a problem any time there's a nonempty intersection between the input and the context collection, and exacerbated by the fact that `.domManip` lies in between the outer method and `buildFragment`.

That said, the `scripts` parameter is probably unnecessary and safe to remove when `buildFragment` is made private. In fact, that's an opportunity to look at more sweeping changes... why exactly do we pluck nodes from the DOM and into a fragment in `buildFragment` instead of `.domManip` anyway? It's an operation that `parseHTML` has to **undo**!